README.md

```diff
@@ -60,11 +60,11 @@ without double processing or missing some elements:
 Pager<Account> accounts = client.listAccounts(new QueryParams());
 
 while (accounts.hasMore()) {
+    System.out.println("Fetching next page...");
+    accounts = accounts.getNextPage();
```

Comment:
Do we still need to overwrite the reference? I think we can remove this right?

Reply:
We can, but should we make `getNextPage()` have a `void` return type to indicate that it shouldn't be used to overwrite the reference?

Reply:
Yes, I think that would be best.